dulwich/cli.py

```diff
@@ -523,12 +523,18 @@ def run(self, args):
 
 class cmd_pack_objects(Command):
     def run(self, args):
-        opts, args = getopt(args, "", ["stdout"])
+        deltify = False
```

Comment:
Ideally we should migrate this to argparse, like some of the other commands in this file.